fission/function.go

```diff
@@ -29,6 +32,37 @@ import (
 	"github.com/fission/fission"
 )
 
+func _fnFetchCode(filePath string) []byte {
```

Comment:
No _ necessary, lowercase function name already means this function is not exported.